### PR TITLE
Small screen instructions

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?
Resolves a design issue with copy codelines on the try landscape page. Makes them more mobile friendly.

#### Where should the reviewer start?
Pull the branch  down and run ```npm install``` 

#### How should this be manually tested?
Open the ```try-landscape.html``` page in your browser / mobile and view the installation process in a small screen.

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
#74 

#### Screenshots (if appropriate)
_n/a_

#### Questions:
_n/a_

### CODE REVIEW

File: features.html [hunk 1]
```diff
@@ -239,7 +241,7 @@ <h3 class="footer__title">Already a customer?</h3>
                     <p class="color--grey"><small>Support hours are as per your service level agreement.</small></p>
                 </div>
                 <div class="twelve-col">
-                    <p><small>&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd Legal information</small></p>
+                    <p><small>&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd <a href="http://www.ubuntu.com/legal?_ga=1.134488433.1025818516.1449589773" class="external">Legal</a> information</small></p>
```
Comment (anthonydillon) [inline features.html]: ga in link and extend the link coverage to include information please.

File: static/js/jquery.command-line.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+          try {
+              var successful = document.execCommand('copy');
+              _gaq.push(['_trackEvent', 'Copy to clipboard', commandInput.get('value'), document.URL]);
+              _this.addClass('is-copied');
+              setTimeout(function(e) {
+                  _this.removeClass('is-copied');
+              }, 300);
+          } catch(err) {
+              console.log('Unable to copy');
```
Comment (anthonydillon) [inline static/js/jquery.command-line.js]: An exception is been thrown here but the execCommand is successful. Can you look into what is erroring here? From the look of it I cant see `_this` is set anywhere.

File: static/scss/objects/_instructions.scss [hunk 3]
```diff
@@ -134,25 +134,36 @@
...
         background-image: url('#{$asset-server-path}v1/994e60f9-get-link-url_16.svg');
         background-repeat: no-repeat;
         background-position: center;
         background-color: $white;
+
+        @media only screen and (min-width : $breakpoint-medium) {
+          width: 50px;
+          right: 0;
+          top: 0;
+        }
```
Comment (anthonydillon) [inline static/scss/objects/_instructions.scss]: :heart:

File: try-landscape.html [hunk 4]
```diff
@@ -287,11 +289,12 @@ <h3 class="footer__title">Already a customer?</h3>
                     <p class="color--grey"><small>Support hours are as per your service level agreement.</small></p>
                 </div>
                 <div class="twelve-col">
-                    <p><small>&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd Legal information</small></p>
+                    <p><small>&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd <a href="http://www.ubuntu.com/legal?_ga=1.134488433.1025818516.1449589773" class="external">Legal</a> information</small></p>
```
Comment (anthonydillon) [inline try-landscape.html]: ga in link and extend the link coverage to include information please.

**Comment by anthonydillon** [comment]
Some issues in code

**Comment by anthonydillon** [comment]
A lot of these changes seem to have been done in https://github.com/ubuntudesign/landscape.canonical.com/pull/89

File: static/js/jquery.command-line.js (hunk 2 above)
Comment (richmccartney) [inline static/js/jquery.command-line.js]: Ah yes I think this is an oversight when I was converting from YUI to jQuery. I've resolved this now and removed the un-required code.

**Comment by richmccartney** [comment]
Resolved the console log issue. Weird conversion from when I wrote the YUI plugin into jQuery.

**Pull request merged**